Navtor_intern_task/FleetManagementAppTest/ShipTest.cs

```diff
@@ -0,0 +1,35 @@
+using FleetManagementApp;
+
+namespace FleetManagementAppTest;
+
+public class ShipTest
+{
+    [Fact]
+    public void Constructor_InvalidId_ThrowsArgumentException()
```

Comment:
Field validation tests should be more explicit. Maybe each field should have it's own static validator, which can then be easily tested in isolation without Ship ctor needed?